Tasks/PowerShellOnTargetMachines2/PowerShellOnTargetMachines.ps1

```diff
@@ -0,0 +1,77 @@
+Trace-VstsEnteringInvocation $MyInvocation
+$global:ErrorActionPreference = 'Continue'
+$global:__vstsNoOverrideVerbose = $true
```

Comment:
You might have to create a new V2 folder aligning with recent changes for other tasks...